<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,3">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QListWidget" name="wordList">
           <property name="font">
            <font>
             <pointsize>18</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,10,1">
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,5,1">
                <item>
                 <widget class="QPushButton" name="leftButton">
                  <property name="text">
                   <string>&lt;&lt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="wordLabel">
                  <property name="font">
                   <font>
                    <pointsize>22</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="rightButton">
                  <property name="text">
                   <string>&gt;&gt;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_6" native="true">
             <widget class="QLabel" name="label_6">
              <property name="geometry">
               <rect>
                <x>210</x>
                <y>30</y>
                <width>54</width>
                <height>17</height>
               </rect>
              </property>
              <property name="text">
               <string>Can use</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_7">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>0</y>
                <width>81</width>
                <height>17</height>
               </rect>
              </property>
              <property name="text">
               <string>Don't know</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_4">
              <property name="geometry">
               <rect>
                <x>160</x>
                <y>0</y>
                <width>41</width>
                <height>21</height>
               </rect>
              </property>
              <property name="text">
               <string>I know</string>
              </property>
             </widget>
             <widget class="QSlider" name="assesment">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>20</y>
                <width>251</width>
                <height>16</height>
               </rect>
              </property>
              <property name="maximum">
               <number>3</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
             <widget class="QLabel" name="label_3">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>30</y>
                <width>71</width>
                <height>17</height>
               </rect>
              </property>
              <property name="text">
               <string>I think I know</string>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_5" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <property name="spacing">
                 <number>1</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="meaningP">
                  <property name="text">
                   <string>Play</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="meaningR">
                  <property name="text">
                   <string>Record</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pronounce">
                  <property name="text">
                   <string>Speak</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
